.travis.yml

```diff
@@ -4,10 +4,22 @@ node_js:
   - "6"
```

Comment:
Could we add version `'node'` here?  This will tests against latest version.  Updating it in all other repos, but I don't want merge conflict here.